addons/mail/static/src/new/web/chatter.scss

```diff
@@ -6,7 +6,7 @@
 }
 
 .o-mail-chatter-topbar {
-    padding-inline-start: $o-horizontal-padding;
+    padding-inline-start: var(--ChatterTopbar-padding-left, 0);
```

Comment:
This is an old variable, not in `new/`.
Also I don't see purpose of using this variable. It's basically the same as `$o-horizontal-padding` (small exception in document chatter, but I'm pretty sure you didn't make the change for that use-case)

Reply:
Actually they are not same. The CSS properties can change by JS but Sass variables are replaced with their actual values at compile time.